test/transcoding/kernel_arg_name.ll

```diff
@@ -1,12 +1,12 @@
-; RUN: llvm-as %s -o - | llvm-spirv -o %t.spv
+; RUN: llvm-as %s -o - | llvm-spirv -preserve-ocl-kernel-arg-type-metadata-through-string -o %t.spv
```

Comment:
Would any part of this test be valid without `-preserve-ocl-kernel-arg-type-metadata-through-string`?

Either the filename should be changed to make it clear that this is testing nonstandard functionality, or an extra RUN line should be added with the CHECKs split to allow testing to be conditionalised.

Reply:
This part of the change has simply been reverted without any rationale given.  Please ensure you answer review commits.  If preservation of the kernel_arg_name metadata is now disabled by default, then why does this have neither a flag nor an explanation?  Was the flag ever needed in the first place, here?

Reply:
Thank you. It seems to me that it would be good to use a single option for all nonstandard kernel arg metadata, at some point... although better still to add an extension, and use that instead (if enabled).